packages/app/src/plugins/cost-insights/ExampleCostInsightsClient.ts

```diff
@@ -116,19 +117,33 @@ export class ExampleCostInsightsClient implements CostInsightsApi {
     return projects;
   }
 
-  async getGroupDailyCost(
-    group: string,
+  async getMetricData(
     metric: string | null,
     intervals: string,
-  ): Promise<Cost> {
+  ): Promise<MetricData> {
     const aggregation = aggregationFor(
       durationOf(intervals),
-      metric ? 0.3 : 8_000,
+      100_000,
+    ).map(entry => ({ ...entry, amount: Math.round(entry.amount) }));
+
+    const cost: MetricData = await this.request(
```

Comment:
This entire thing is getting big, wanna move it over to the plugin? :grin:

When it's on the app it's not easily usable by people to try out in the own deployment or demo deployment